operator/gitops/argocd/grafana/dashboards/pipeline-service-dashboard.json

```diff
@@ -1529,12 +1435,481 @@
               "refId": "A"
             }
           ],
-          "title": "Average scheduling duration",
+          "title": "Average time for learning of pipelinerun creation",
```

Comment:
this is where I start adding the new panels to the now named "peformance analysis" row